mowgli_etl/pipeline/wdc/wdc_heuristic_product_type_classifier.py

```diff
@@ -0,0 +1,53 @@
+from spacy import load
+
+from mowgli_etl.pipeline.wdc.wdc_product_type import WdcProductType
+from mowgli_etl.pipeline.wdc.wdc_product_type_classifier import WdcProductTypeClassifier
+
+
+class WdcHeuristicProductTypeClassifier(WdcProductTypeClassifier):
+    def classify(self, *, title: str) -> WdcProductType:
```

Comment:
Here you could use the "input model" NamedTuple/@dataclass again, since the heuristics will want to consult other bits of the WDC json model.